source/includes/api/_dataset.md

```diff
@@ -614,6 +614,8 @@ curl -X PATCH https://api.resourcewatch.org/v1/dataset/<dataset-id> \
     This is an authenticated endpoint!
 </aside>
 
+Note: When updating the `env` property of a dataset this way the change will cascade down to all associated layer and widget entities.
```

Comment:
is it me, or this sentence sounds weird?

Reply:
It does, fixed!